packages/platform/atoms/event-types/wrappers/EventTypePlatformWrapper.tsx

```diff
@@ -63,17 +64,34 @@ const EventType = ({
   const [slugExistsChildrenDialogOpen, setSlugExistsChildrenDialogOpen] = useState<ChildrenEventType[]>([]);
   const { data: user, isLoading: isUserLoading } = useMe();
 
+  const handleDeleteSuccess = () => {
+    showToast(t("event_type_deleted_successfully"), "success");
```

Comment:
Since we added it to `AvailabilitySettings` in [this PR](https://github.com/calcom/cal.com/pull/17216), maybe we can also add `disableToasts` optional param to `AvailabilitySettings` and with time add them to rest of components to allow more flexiblity to devs.